definitions/package-lock.json

```diff
@@ -0,0 +1,2627 @@
+{
```

Comment:
Remove this file, we use yarn for our lock files.

Reply:
Let me go ahead and do that

Reply:
@GAntoine removed the package-lock file and updated the comment to reflect the affected spec.